knowledge-base/font-icons-not-rendering.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Font Icons do not render in Telerik UI for Blazor 4.6
+description: Font Icons do not render in Telerik UI for Blazor 4.6
+type: troubleshooting
+page_title: Font Icons do not Render in Telerik UI for Blazor 4.6.0
+slug: icon-kb-font-icons-not-rendering
+position: 
+tags: telerik, blazor, fonticon, font, icon
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>4.6</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>Progress® Telerik® UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+After upgrading to the 4.6.0 version of the Telerik UI for Blazor suite the Font Icons in my application are not longer showing (rendering). 
```

Comment:
Link the font icons article somewhere in the article content and also add it as a See Also item.

Consider linking this KB from the Font Icons page and Rendering Changes for more incoming traffic. Customers may refer to the existing browsable articles when they hit an issue.

Reply:
It is already linked in the 4.6.0 Rendering changes article (another PR currently). Right now I try to think of the best possible place in the Icons article to link the KB

Reply:
Added  a link to the KB in the Icons article